src/backend/distributed/citus--6.1-10--6.1-11.sql

```diff
@@ -0,0 +1,66 @@
+/* citus--6.1-10--6.1-11.sql */
+
+SET search_path = 'pg_catalog';
+
+DROP FUNCTION master_drop_sequences(text[], text, bigint);
+
+CREATE FUNCTION master_drop_sequences(sequence_names text[])
+    RETURNS void
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$master_drop_sequences$$;
+COMMENT ON FUNCTION master_drop_sequences(text[])
+    IS 'drop specified sequences from the cluster';
+
+CREATE OR REPLACE FUNCTION pg_catalog.citus_drop_trigger()
+    RETURNS event_trigger
+    LANGUAGE plpgsql
+    SECURITY DEFINER
+    SET search_path = pg_catalog
+    AS $cdbdt$
+DECLARE
+    v_obj record;
+    sequence_names text[] := '{}';
+    table_colocation_id integer;
+    propagate_drop boolean := false;
+BEGIN
+    -- collect set of dropped sequences to drop on workers later
+    SELECT array_agg(object_identity) INTO sequence_names
+    FROM pg_event_trigger_dropped_objects()
+    WHERE object_type = 'sequence';
+   
+    FOR v_obj IN SELECT * FROM pg_event_trigger_dropped_objects() LOOP
+        IF v_obj.object_type NOT IN ('table', 'foreign table') THEN
+           CONTINUE;
+        END IF;
```

Comment:
Just for consideration, since we're redefining, in MX we had rewritten these IF statements a bit:
https://github.com/citusdata/citus-mx/blob/masterless/main/src/backend/distributed/citus--5.2-1--5.2-mx.sql#L267